gsoc/background.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding internship program funded by 
+Google. Google distributes funds to open software development organizations to 
+recruit new developers and mentors for summer internships. You get paid to write 
+software at home (or wherever you like), with guidance from expert mentors 
+distributed around the globe.
+
+To apply to be a GSoC intern, you must write an application to GSoC and an 
+application to OpenAstronomy. Each summer, several projects are offered by 
+potential OpenAstronomy mentors, with well-defined scope and pre-requisite 
+experience. If you are selected by both Google and OpenAstronomy, you will be
+a GSoC intern.
+
+### What is OpenAstronomy?
+
+OpenAstronomy is an umbrella organization which acts as an interface between
+the Google Summer of Code and several open source, open development projects
+in the astronomical community, including astropy and sunpy. It coordinates
+mentors and projects for Google Summer of Code internships related to astronomy.
+
+### What is the application process?
+
+You must submit your application to the Google Summer of Code in two parts -- 
+one for Google, and one for your potential project mentor. There's some basic 
+biographical information that Google needs for their part of the application, 
+and we will ask you for more project-specific personal backaground information 
+in the OpenAstronomy application. 
+
+The complete details on applying are in the [GSoC Student Guide].
+
+### Who is eligible to apply to GSoC?
+
+OpenAstronomy is actively seeking applicants who are senior undergraduates or
+junior graduate students with experience developing software for astronomy in 
+Python and other languages. We are looking for students who have or are pursuing
+degrees in astronomy (or physics), who are interested in developing open source 
+software. You don't need to be a Python guru or an astronomical coordinate 
+system expert to be eligible -- projects are available that cover a range of 
+previous experience in Python and/or astronomy. Underrepresented students are 
+especially encouraged to apply.
+
+### Testimonials
+
+> In the summer after my second year of graudate school, I didn't have funding
+to work on any projects in my home department. I applied and was selected for
+GSoC 2015, and I worked with another student and several encouraging mentors on
+developing a new astropy affliated package, called 
+[astroplan](http://astroplan.readthedocs.io/). The internship greatly improved
+my coding style, taught me loads about managing large projects, version control,
+testing, and packaging. Do you have a summer without research funding? Apply!
+
+Brett Morris, GSoC student 2015
```

Comment:
That's brilliant!! Thanks @bmorris3 for sharing it :) - Should we put your picture (or GH avatar) next to it?? Also, if you put your name with a link to your github profile it helps.